src/main/java/com/openmrs/migrator/Migrator.java

```diff
@@ -119,10 +119,23 @@ private void executeSetupCommand() throws IOException, SQLException, SettingsExc
     List<String> pdiFiles = new ArrayList<>();
 
     pdiFiles.add(SettingsService.PDI_RESOURCES_DIR + "/jobs/control-center.kjb");
+    pdiFiles.add(SettingsService.PDI_RESOURCES_DIR + "/jobs/preparation.kjb");
```

Comment:
@ynurmahomed we are moving away from this approach of hard coding the resources in the Java app - better that we design the app to just read all resources contained under `pdiresources/` folder and treat the pdi files as input files.